recipe/meta.yaml

```diff
@@ -12,34 +12,24 @@ source:
 
 build:
   number: 0
-  skip: true  # [win and vc<14]
-
-requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    #- cross-python_{{ target_platform }}     # [build_platform != target_platform]
-  host:
-    - python
-    - pip >=18.1
-    - setuptools >=34.0.0
+  skip: true  # [py<37]
 
 outputs:
   - name: {{ name }}
     build:
-      script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --verbose
-      skip: true  # [win and vc<14]
+      script: python -m pip install . --no-deps --ignore-installed --verbose
     requirements:
-      build:
-        - python                                 # [build_platform != target_platform]
-        #- cross-python_{{ target_platform }}     # [build_platform != target_platform]
       host:
         - python
-        - pip >=18.1
-        - protobuf >=3.12.0
```

Comment:
as far as I can tell there is no compilation taking place so no need for a full complement of host requirements